mailclient.py

```diff
@@ -20,11 +20,115 @@
+                      "Filter coffee sugar": 8,
+                      "Hot water": 8,
+                      "Latte macchiato": 8,
+                      "Latte macchiato sugar": 8,
+                      "Wiener melange": 8}
+
+milk_powder_usage = {"Café au lait": 8,
+                     "Café au lait sugar": 8,
+                     "Café choco": 0,
+                     "Cappuccino": 8,
+                     "Cappuccino sugar": 8,
+                     "Coffee": 0,
+                     "Coffee milk sugar": 8,
+                     "Coffee with sugar": 0,
+                     "Coffee with milk": 8,
+                     "Chocolate": 0,
+                     "Choco lux": 0,
+                     "Espresso": 0,
+                     "Espresso with milk": 8,
+                     "Filter kaffe": 0,
+                     "Filter koffie": 0,
+                     "Filter coffee milk": 8,
+                     "Filter coffee milk sugar": 8,
+                     "Filter coffee sugar": 0,
+                     "Hot water": 0,
+                     "Latte macchiato": 8,
+                     "Latte macchiato sugar": 8,
+                     "Wiener melange": 8}
+
+cacao_powder_usage = {"Café au lait": 0,
+                      "Café au lait sugar": 0,
+                      "Café choco": 8,
+                      "Cappuccino": 0,
+                      "Cappuccino sugar": 0,
+                      "Coffee": 0,
+                      "Coffee milk sugar": 0,
+                      "Coffee with sugar": 0,
+                      "Coffee with milk": 0,
+                      "Chocolate": 8,
+                      "Choco lux": 8,
+                      "Espresso": 0,
+                      "Espresso with milk": 0,
+                      "Filter kaffe": 0,
+                      "Filter koffie": 0,
+                      "Filter coffee milk": 0,
+                      "Filter coffee milk sugar": 0,
+                      "Filter coffee sugar": 0,
+                      "Hot water": 0,
+                      "Latte macchiato": 0,
+                      "Latte macchiato sugar": 0,
+                      "Wiener melange": 4}
+
+sugar_usage = {"Café au lait": 0,
+               "Café au lait sugar": 4,
+               "Café choco": 0,
+               "Cappuccino": 0,
+               "Cappuccino sugar": 4,
+               "Coffee": 0,
+               "Coffee milk sugar": 4,
+               "Coffee with sugar": 4,
+               "Coffee with milk": 0,
+               "Chocolate": 0,
+               "Choco lux": 0,
+               "Espresso": 0,
+               "Espresso with milk": 0,
+               "Filter kaffe": 0,
+               "Filter koffie": 0,
+               "Filter coffee milk": 0,
+               "Filter coffee milk sugar": 4,
+               "Filter coffee sugar": 4,
+               "Hot water": 0,
+               "Latte macchiato": 0,
+               "Latte macchiato sugar": 4,
+               "Wiener melange": 4}
+
```

Comment:
Pull these out into a discrete file, maybe?

Reply:
Added a seperate file for this in 040fbc32cbd6182f1f1b4a7572812778e3afe455. Also added max values into this file